sections/futures/TradeCrossMargin/CrossMarginInfoBox.tsx

```diff
@@ -26,23 +30,28 @@ import {
 } from 'utils/formatters/number';
 
 import EditLeverageModal from './EditLeverageModal';
+import WithdrawKeeperDepositModal from './WithdrawKeeperDepositModal';
 
 type Props = {
 	editingLeverage?: boolean;
 };
 
 function MarginInfoBox({ editingLeverage }: Props) {
+	const { keeperEthBal } = useCrossMarginKeeperDeposit();
```

Comment:
Doesn't look like this value is refetched when things change. We should fetch it if a user submits an order which deposits ETH, or if they withdraw their ETH.